mods/gennext/README.md

```diff
@@ -236,6 +237,14 @@ New mechanic: Intrinsics:
 
   - Mismagius: Cursed Body
 
+  - Rotom:
+    - Normal: Motor Drive
+    - Heat: Flash Fire
+    - Wash: Hydration
+    - Mow: Sap Sipper
+    - Fan: Gale Wings
```

Comment:
I can't think of any other move that fits its existence as a fan, though. Unless we throw it under the bus with Cloud Nine or Air Lock, which also means it can't spam Hurricane on rain teams.